src/chapters/07-reusable-components.md

```diff
@@ -328,9 +353,9 @@ Note that we did not mark our getter as `@tracked`. Unlike instance variables, g
 
 That being said, the values _produced_ by getters can certainly change. In our case, the value produced by our `src` getter depends on the values of `lat`, `lng`, `width`, `height` and `zoom` from `this.args`. Whenever these *[dependencies][TODO: link to dependencies]* get updated, we would expect `{{this.src}}` from our template to be updated accordingly.
 
-Ember does this by automatically tracking any variables that were accessed while computing a getter's value. As long as the dependencies themselves are marked as `@tracked`, Ember will know exactly when to invalidate and re-render any templates that may potentially contain any "stale" (outdated) getter values. This feature is also know as *[auto-track][TODO: link to auto-track]*.
+Ember does this by automatically tracking any variables that were accessed while computing a getter's value. As long as the dependencies themselves are marked as `@tracked`, Ember will know exactly when to invalidate and re-render any templates that may potentially contain any "stale" (outdated) getter values. This feature is also know as *[auto-track][TODO: link to auto-track]*. Since all arguments (`this.args.*`) are implicitly marked as `@tracked` by the Glimmer component superclass that we inherited from, everything Just Works&trade;.
```

Comment:
```suggestion
Ember does this by automatically tracking any variables that were accessed while computing a getter's value. As long as the dependencies themselves are marked as `@tracked`, Ember knows exactly when to invalidate and re-render any templates that may potentially contain any "stale" and outdated getter values. This feature is also known as *[auto-track][TODO: link to auto-track]*. All arguments that can be accessed from `this.args` (in other words, `this.args.*`) are implicitly marked as `@tracked` by the Glimmer component superclass. Since we inherited from that superclass, everything Just Works&trade;.
```

Reply:
Can feel free to tweak this. This edit is more to clarify what `this.args.*` is, since I'm not sure that beginners would necessarily know that it refers to "all arguments" if they hadn't seen that syntax before.

Reply:
Seems good. Makes it easier to `ctrl+f` too.